runtime/onert/api/nnfw/src/nnfw_internal.cc

```diff
@@ -48,7 +48,7 @@ NNFW_STATUS nnfw_load_circle_from_buffer(nnfw_session *session, uint8_t *buffer,
 NNFW_STATUS nnfw_load_model_from_modelfile(nnfw_session *session, const char *file_path)
```

Comment:
Is there any reason to maintain the name(`nnfw_load_model_from_modelfile`)?

Reply:
`onert_run` and other test drivers are using this API. It will be removed after migration.